gnomad_qc/v3/create_release/create_hgdp_tgp_subset.py

```diff
@@ -1768,9 +628,10 @@ def main(args):
         mt = adjust_subset_alleles(mt)
 
         logger.info(
-            "Note: for the finalized HGDP + TGP subset frequency, dense MT, and VCFs we adjust the sex genotypes and add "
-            "a fix for older GATK gVCFs with a known depletion of homozygous alternate alleles, and remove standard "
-            "GATK LowQual variants and variants in centromeres and telomeres."
+            "Note: this sparse MT is the raw subset MT and therefore it does not have adjusted sex genotypes and "
```

Comment:
```suggestion
            "NOTE: This sparse MT is the raw subset MT and therefore it does not have adjusted sex genotypes and "
```
Adjusting capitalizations for consistency within the script